From bb804d208509ecc0ae4092f3e48d4cdfbbe9b027 Mon Sep 17 00:00:00 2001
From: Patrick Delaunay <patrick.delaunay@foss.st.com>
Date: Fri, 6 Jan 2023 13:20:17 +0100
Subject: [PATCH 087/117] configs: stm32mp13: Activate CONFIG_CMD_FUSE

Activate the command fuse to access on STM32MP13x OTP with
the BSEC driver.

[Backport of commit 7de0d10ca716 ("configs: stm32mp13: Activate
 CONFIG_CMD_FUSE")]

Signed-off-by: Patrick Delaunay <patrick.delaunay@foss.st.com>
Reviewed-by: Patrice Chotard <patrice.chotard@foss.st.com>
Change-Id: I582a6a1b443ccf0fda42be911f52e814d4ddb616
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/u-boot/+/270217
---
 configs/stm32mp13_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/configs/stm32mp13_defconfig b/configs/stm32mp13_defconfig
index 26beb73e84..c40b0e8b68 100644
--- a/configs/stm32mp13_defconfig
+++ b/configs/stm32mp13_defconfig
@@ -26,6 +26,7 @@ CONFIG_CMD_MEMINFO=y
 CONFIG_CMD_MEMTEST=y
 CONFIG_CMD_UNZIP=y
 CONFIG_CMD_CLK=y
+CONFIG_CMD_FUSE=y
 CONFIG_CMD_GPIO=y
 CONFIG_CMD_I2C=y
 CONFIG_CMD_LSBLK=y
-- 
2.39.2

